graspologic/inference/latent_distribution_test.py

```diff
@@ -39,6 +51,10 @@ class LatentDistributionTest(BaseInference):
 
     Parameters
     ----------
+    A1, A2 : variable (see description)
```

Comment:
you say see description, but you didn't actually copy the description which talks about passing embeddings or passing graphs.